src/main/java/codes/biscuit/skyblockaddons/mixins/MixinPlayerControllerMP.java

```diff
@@ -34,6 +34,15 @@ private void onPlayerDamageBlock(BlockPos loc, EnumFacing face, CallbackInfoRetu
             EntityPlayerSP p = mc.thePlayer;
             ItemStack heldItem = p.getHeldItem();
             Block block = mc.theWorld.getBlockState(loc).getBlock();
+            if (heldItem != null && block.equals(Blocks.reeds)) {
```

Comment:
I suggest avoiding repeating code with this: 

```suggestion
        if (heldItem != null) {
        
        if (block.equals(Blocks.melon_stem) || block.equals(Blocks.pumpkin_stem)
        || (block.equals(Blocks.reeds) && mc.theWorld.getBlockState(loc.down()).getBlock() != Blocks.reeds) {
          
          cir.setReturnValue(false);
          if (System.currentTimeMillis() - lastMessage > 15000) {
            
            lastMessage = System.currentTimeMillis();
            
            main.getUtils().sendMessage(EnumChatFormatting.RED
              + Message.MESSAGE_CANCELLED_STEM_BREAK.getMessage());
          }
        }
      }
```
please disregard formatting

Reply:
Yeah, super basic change I should have caught, was quite late when I wrote this. my bad, thanks for finding!